pom.xml

```diff
@@ -32,5 +40,10 @@
 		<artifactId>spring-context</artifactId>
 		<version>4.3.3.RELEASE</version>
 	</dependency>
+        <dependency>
```

Comment:
Plugin flyway-maven-plugin ma dependency na H2, przenieś to dependency do sekcji dependencies poza ten plugin - w ten sposób biblioteka H2 będzie dostępna w ramach całego projektu, a nie tylko plugina flyway-maven-plugin